CHANGELOG.md

```diff
@@ -286,71 +259,4 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
-- Do not return picture content on create [(#292)](https://github.com/ManageIQ/manageiq-api/pull/292)
-- Fix special characters in MIQ_GROUP header [(#287)](https://github.com/ManageIQ/manageiq-api/pull/287)
-- Remove groups from editable attributes for users [(#286)](https://github.com/ManageIQ/manageiq-api/pull/286)
-- Updating the API so it now returns nil attributes. [(#253)](https://github.com/ManageIQ/manageiq-api/pull/253)
-- add image scanning endpoint [(#245)](https://github.com/ManageIQ/manageiq-api/pull/245)
-
-## Unreleased as of Sprint 77 ending 2018-01-15
-
-### Added
-- Add Settings API for servers and regions [(#275)](https://github.com/ManageIQ/manageiq-api/pull/275)
-
-### Fixed
-- Update role identifiers for cloud subnets subcollection [(#280)](https://github.com/ManageIQ/manageiq-api/pull/280)
-- Update dialog copy product feature [(#262)](https://github.com/ManageIQ/manageiq-api/pull/262)
-- Fix regression for expansion of subcollection on a resource [(#261)](https://github.com/ManageIQ/manageiq-api/pull/261)
-- Squeeze consecutive slashes in the path portion of the URI [(#228)](https://github.com/ManageIQ/manageiq-api/pull/228)
-
-## Unreleased as of Sprint 76 ending 2018-01-01
-
-### Added
-- Allow assigning/un-assigning of alert definitions to alert profiles [(#149)](https://github.com/ManageIQ/manageiq-api/pull/149)
-
-## Unreleased as of Sprint 75 ending 2017-12-11
-
-### Added
-- Generic Object Definition OPTIONS: send hashes rather than arrays [(#238)](https://github.com/ManageIQ/manageiq-api/pull/238)
-- Use type names directly from GenericObjectDefinition object [(#232)](https://github.com/ManageIQ/manageiq-api/pull/232)
-- Add delete for router [(#193)](https://github.com/ManageIQ/manageiq-api/pull/193)
-
-## Unreleased as of Sprint 74 ending 2017-11-27
-
-### Added
-- Add subresource_action_identifier spec helper [(#225)](https://github.com/ManageIQ/manageiq-api/pull/225)
-- Adding support for optional token_ttl in the UserTokenService. [(#214)](https://github.com/ManageIQ/manageiq-api/pull/214)
-- Adding custom_actions support for /api/cloud_networks [(#200)](https://github.com/ManageIQ/manageiq-api/pull/200)
-- Add support for /api/switches [(#191)](https://github.com/ManageIQ/manageiq-api/pull/191)
-- Add support for /api/container_volumes [(#190)](https://github.com/ManageIQ/manageiq-api/pull/190)
-- Add support for /api/container_templates [(#188)](https://github.com/ManageIQ/manageiq-api/pull/188)
-- Adding support for /api/container_images [(#185)](https://github.com/ManageIQ/manageiq-api/pull/185)
-- Adding support for /api/container_groups [(#184)](https://github.com/ManageIQ/manageiq-api/pull/184)
-- Adding support for /api/cloud_object_store_containers [(#183)](https://github.com/ManageIQ/manageiq-api/pull/183)
-- Add custom action support for models already exposed in api that need it [(#163)](https://github.com/ManageIQ/manageiq-api/pull/163)
-
-## Unreleased as of Sprint 72 ending 2017-10-30
```

Comment:
This `Sprint 72 ending 2017-10-30` section should be a part of Gaprindashvili release, as we branched Gaprindashvili on Oct 31.

Reply:
thanks Satoe. Corrected now.